primitives/native-token-management/src/lib.rs

```diff
@@ -0,0 +1,117 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "std")]
+pub use inherent_provider::*;
+
+use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use sidechain_domain::*;
+use sp_inherents::*;
+use sp_runtime::{scale_info::TypeInfo, traits::Block as BlockT};
+
+#[cfg(test)]
+mod tests;
+
+pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"ntreserv";
```

Comment:
Please add a comment why `ntreserv`

Reply:
It came from 'native token reserve', but it's not just connected to the reserve. Anywya, I renamed it to `nattoken` so it's clearer.